src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,26 +38,29 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            App.currProject = project;
```

Comment:
This isn't a bad idea, but I think we need to be careful about adding a new global var. 

Couple of thoughts:

1. Consider renaming to `currentProject`.
2. Add a default value and documentation to the `App` class.
3. We need to be careful about this becoming stale. For example, switching between views or switching projects should never contain a reference to the wrong project.